src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -0,0 +1,106 @@
+==================
+Input Deck Options

```

Comment:
These titles don't coincide with the table or part of the tables that they go with.  Something like:

===========
thermal_solver
===========

------
mesh
------
.. list-table:: Fields
   :widths: 25 25 25 25 25
   :header-rows: 1
   :stub-columns: 1

   * - Field Name
     - Description
     - Default Value
     - Range
     - Required
   * - filename
     - file for thermal solver
     - Not specified
     - Not specified
     - True
   * - serial
     - serial value
     - Not specified
     - Not specified
     - Not specified
   * - parallel
     - parallel value
     - Not specified
     - Not specified
     - True

-----
next table
-----

Reply:
So you're saying to split the fields into several tables under their respective inlet::Table name?

Reply:
How would the code handle nested Tables?  Or arrays
E.g.

```lua
foo { 
  foo_a = 1, 
  foo_b = nested_bar = {
      bar_a = "hello",
      bar_b = true,
      doubly_nested = {
          baz = "fun",
      }
  },
  foo_c = 3.14,
  foo_d = {
     [1] = "the first",
     [2] = "the_second",
     [3] = "etc...",
}
```

Reply:
The original code I pushed would create one large table for all fields, but I'll be pushing an update that'll put the fields in a chart under the label of the lowest inlet::Table containing those fields.

Reply:
As for arrays, I didn't know you could have arrays in lua so I'll have to look into it